assets/base.css

```diff
@@ -2462,3 +2462,46 @@ details-disclosure > details {
   height: 0;
   padding-bottom: var(--ratio-percent);
 }
+
+.content-containers {
```

Comment:
Extremely nitpicky but I don't think the class name should be plural. content-containers sounds like a wrapper element meant to have children of type content-container.